docs/guides/algorithmiq-tem.ipynb

```diff
@@ -175,7 +175,9 @@
     "service = QiskitRuntimeService()\n",
     "backend_name = service.least_busy(operational=True).name\n",
     "\n",
-    "instance = \"<IQP_HUB/IQP_GROUP/IQP_PROJECT>\"\n",
+    "# Instance is a string of the form \"<IQP_HUB/IQP_GROUP/IQP_PROJECT>\".\n",
+    "# The following line gets the instance of the active QiskitRuntimeService account.\n",
+    "instance = service.active_account()[\"instance\"]\n",
```

Comment:
I see that in the rest of the notebooks we hide this line from the users. Do we want to show it here?

Reply:
No strong preference, I think I left this one in because it's already using `QiskitRuntimeService`, whereas the other notebooks didn't.

Reply:
Alternatively, we could un-hide the other cells and add a similar comment

Reply:
Oh, I see. Then, I think we can leave it as is :+1: